.gitignore

```diff
@@ -1,4 +1,5 @@
-/.composer.lock
+/composer.lock
 /build/
 /consul-configuration/data-dir/
 /vendor/
+/.idea
```

Comment:
This one should be removed. This should be in your personal global ignore list instead.